README.md

```diff
@@ -19,7 +19,7 @@ And for rendering add `<NextNProgress />` inside `Container` component.
 ```
 <NextNprogress
   color="#29D"
-  startPosition="0.3"
+  startPosition={0.3}
   stopDelayMs="200"
```

Comment:
For me, `stopDelayMs` is also causing the same effect if I use string. The progress bar shows up only after page load. We can also make this a number.

Reply:
Sounds good- will update the request now